pom.xml

```diff
@@ -24,7 +24,7 @@
         <maven.resources.encoding>${project.build.sourceEncoding}</maven.resources.encoding>
         <version.maven>3.6.0</version.maven>
         <!-- dependency versions should go here, so below they can be omitted -->
-        <version.dependency.testng>7.0.0</version.dependency.testng>
+        <version.dependency.testng>7.1.0</version.dependency.testng>
```

Comment:
You can use https://mvnrepository.com/ for checking dependency version. Just if you don't know about it.

Reply:
You mean, literally by looking for new one on the website? I thought that I actually got this one version from the webside, but probably it was downloaded automatically by IDE. Also, mvn versions:display-dependency-updates is useful and that's why I updated it:)